src/spryker/application/skeleton/config/install/docker.yml.twig

```diff
@@ -220,7 +220,7 @@ sections:
 
     queue-worker:
         run:
-            command: "vendor/bin/console queue:worker:start -vvv"
+            command: "vendor/bin/console queue:worker:start --stop-when-empty -vvv"
```

Comment:
This flag was already there, but there was an issue that it wasn't processing all pending messages (sometimes). Tho they might have fixed it in January release.

Reply:
Something we can test with behat to save 9 minutes?

Reply:
It's random, so can't really test!

But if you can make the CI build green, we can try to merge it and see how it goes. If it causes issue, can revert it.

Reply:
Build isn't green due to an error deleting php docker images, but the spryker part is all okay :)